app/server.R

```diff
@@ -343,19 +343,29 @@ server <- function(input, output, session) {
     updateAsOfChoices(session, truthDf)
 
     # Format and transform data for plot
-    filteredScoreDf <- filteredScoreDf %>%
-      filter(!is.na(Week_End_Date)) %>%
-      select(Forecaster, Forecast_Date, Week_End_Date, Score, ahead) %>%
-      mutate(across(where(is.numeric), ~ round(., 2)))
+    filteredScoreDf <- filter(
+      filteredScoreDf[, c("Forecaster", "Forecast_Date", "Week_End_Date", "Score", "ahead")],
+      !is.na(Week_End_Date)
+    )
+    filteredScoreDf$Score <- round(filteredScoreDf$Score, 2)
```

Comment:
extra explanation for future travelers:

This change is equivalent.

`Score` is the only column where rounding to 2 decimal places will have any effect.

`ahead` is numeric but [takes integer values](https://github.com/cmu-delphi/forecast-eval/blob/dcfcde4d94597a6f5294a5e8af2950db59ca2b28/app/global.R#L28-L32).